src/NetAnalyzers/UnitTests/Microsoft.CodeQuality.Analyzers/QualityGuidelines/UseLiteralsWhereAppropriateTests.cs

```diff
@@ -183,12 +183,33 @@ public class Class1
 {
 	internal static readonly IntPtr field1 = (nint)0;
 	internal static readonly UIntPtr field2 = (nuint)0;
-    internal static readonly nint field3 = (nint)0;
-	internal static readonly nuint field4 = (nuint)0;
 }",
             }.RunAsync();
         }
 
+        [Fact]
+        public async Task CA1802_CSharp_nint_nuint_Diagnostic()
```

Comment:
Test the fixer as well?

Reply:
I am missing all edges today :)

Reply:
📝 Not sure I ever mentioned this in the past, but it's not necessary to test diagnostics and fixers _separately_. Adding the expected result to this test would cover it perfectly.

Reply:
Yes you did and I also think that's better than to duplicate tests.